src/transforms/repair.jl

```diff
@@ -82,7 +82,7 @@ function apply(::Repair{8}, poly::PolyArea)
 end
 
 function apply(::Repair{8}, poly::Ngon)
-  v = poly |> vertices |> repair8
+  v = poly |> vertices |> collect |> repair8
```

Comment:
no need to collect in case of repair8 with Ngon. We need to generalize the repair8 function to receive iterables.